api/tests/unit/environments/test_unit_environments_views_sdk_environment.py

```diff
@@ -26,19 +27,28 @@
     from organisations.models import Organisation
 
 
+@pytest.mark.parametrize(
+    "use_v2_feature_versioning, total_queries", [(True, 16), (False, 15)]
+)
 def test_get_environment_document(
     organisation_one: "Organisation",
     organisation_two: "Organisation",
     organisation_one_project_one: "Project",
     django_assert_num_queries: "DjangoAssertNumQueries",
+    use_v2_feature_versioning: bool,
+    total_queries: int,
 ) -> None:
     # Given
     project = organisation_one_project_one
     project2 = Project.objects.create(
         name="standin_project", organisation=organisation_two
     )
 
-    environment = Environment.objects.create(name="Test Environment", project=project)
+    environment = Environment.objects.create(
+        name="Test Environment",
+        project=project,
+        use_v2_feature_versioning=use_v2_feature_versioning,
+    )
```

Comment:
I don't think this will matter, because we haven't yet created any features, but just for completeness, I recommend doing: 

```suggestion
    environment = Environment.objects.create(
        name="Test Environment",
        project=project,
    )
    enable_v2_versioning(environment.id)
```

Reply:
Done